db/week-3/lesson.md

```diff
@@ -21,6 +21,18 @@ Read the Mentors Notes [here](./mentors.md)
 
 ---
 
+## Learning Objectives
+
+By the end of this lesson students should be able to
+
+- Create an API endpoint that can add data to a database
+- Write code that effectively validates data before adding it to a database
```

Comment:
```suggestion
- Write code that effectively validates data before adding it to a database
- Be aware of the dangers of SQL injection attacks, and how to avoid them
```